src/Agda2Hs/Compile.hs

```diff
@@ -24,81 +25,67 @@ import Agda2Hs.Compile.Types
 import Agda2Hs.Compile.Utils ( setCurrentRangeQ, tellExtension )
 import Agda2Hs.Pragma
 
--- Needs a list of rewrite rules too.
+
 initCompileEnv :: TopLevelModuleName -> SpecialRules -> CompileEnv
 initCompileEnv tlm rewrites = CompileEnv
-  { currModule = tlm
-  , minRecordName = Nothing
-  , locals = []
+  { currModule        = tlm
```

Comment:
It would be a bit nicer to bundle all cosmetic changes such as this one in a separate PR, since it will make it easier to locate any bugs that might be introduced.